qa/integration/src/test/resources/features/job/JobTargetsServiceI9n.feature

```diff
@@ -9,9 +9,10 @@
 # Contributors:
 #     Eurotech - initial API and implementation
 ###############################################################################
-@integration
 @jobs
 @jobTargetService
+@integration
```

Comment:
Same comment as above; if you wanted to put "integration" tags in the end it's ok, otherwise put it back.